staging/src/k8s.io/apimachinery/pkg/runtime/serializer/json/json.go

```diff
@@ -157,7 +167,7 @@ func (s *Serializer) Decode(originalData []byte, gvk *schema.GroupVersionKind, i
 		types, _, err := s.typer.ObjectKinds(into)
 		switch {
 		case runtime.IsNotRegisteredError(err), isUnstructured:
-			if err := jsoniter.ConfigCompatibleWithStandardLibrary.Unmarshal(data, into); err != nil {
```

Comment:
looks like there are four other uses of `jsoniter.ConfigCompatibleWithStandardLibrary` in the codebase... should replace those as well, and think about how to ensure we only use a standard consistent config (probably something like https://github.com/kubernetes/kubernetes/pull/63284)

Reply:
Replaced the other references in the latest commit.

#63284 is a good follow-up.